Source/SwiftLintFrameworkTests/ConfigurationTests.swift

```diff
@@ -6,7 +6,8 @@
 //  Copyright © 2015 Realm. All rights reserved.
 //
 
-import SwiftLintFramework
+@testable import SwiftLintFramework
```

Comment:
I think that's probably ok. I get slightly nervous with equality stuff because it's easy for someone to add a member to the respective type without including it in the member-wise check. You can somewhat mitigate this by using Swift's limited reflection to write a test to assert that the number of members hasn't changed. But I often just keep this stuff out of public API until the data structures have stabilized or it's otherwise needed.